app/asset-manifest.js

```diff
@@ -1,13 +1,22 @@
 import environment from './config/environment';
 
-const metaName = environment.modulePrefix + '/asset-manifest';
+const modulePrefix = environment.modulePrefix;
+const metaName = `${modulePrefix}/asset-manifest`;
+const nodeName = `${modulePrefix}/node-asset-manifest`;
+
 let config = {};
 
 try {
-  const rawConfig = document.querySelector('meta[name="' + metaName + '"]').getAttribute('content');
-  config = JSON.parse(unescape(rawConfig));
+  // If we have a Node version of the asset manifest, use that for FastBoot and
+  // similar environments.
+  if (require.has(nodeName)) {
+    config = require(nodeName).default; // eslint-disable-line
+  } else {
+    const rawConfig = document.querySelector('meta[name="' + metaName + '"]').getAttribute('content');
+    config = JSON.parse(unescape(rawConfig));
+  }
 } catch(err) {
-  throw new Error('Could not read asset manifest from meta tag with name "' + metaName + '".');
+  throw new Error('Could not load asset manifest from meta tag with name "' + metaName + '".');
```

Comment:
Will update this depending on the outcome of https://github.com/trentmwillis/ember-asset-loader/pull/23#discussion_r79737254